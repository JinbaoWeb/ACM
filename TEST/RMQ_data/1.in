5 3
180 179 78 65 22
1 3
1 4
2 5
