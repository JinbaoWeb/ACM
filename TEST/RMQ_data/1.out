78
180
65
180
22
179
